.github/workflows/windows.yml

```diff
@@ -10,38 +10,44 @@ jobs:
     runs-on: windows-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Cache Nasm install
-      uses: actions/cache@v1
-      id: cache-nasm
-      env:
-        cache-name: cache-nasm
-      with:
-        path: "C:\\Program Files\\Nasm"
-        key: cache-nasm-v1
-        restore-keys: cache-nasm-v1
```

Comment:
`NASM` is already on `OpenSSL`?

Reply:
I have no idea if is it or not, but at least vcpkg's openssl doesn't require to priorly explicitly install nasm.

Reply:
It _probably_ installs internally (at least it installs strawberry-perl), but...

Reply:
haha, okay then, if it works :D